pkg/controller/accountclaim/reuse.go

```diff
@@ -185,7 +189,55 @@ func (r *ReconcileAccountClaim) cleanUpAwsAccount(reqLogger logr.Logger, awsClie
 	return nil
 }
 
-func (r *ReconcileAccountClaim) cleanUpAwsAccountSnapshots(reqLogger logr.Logger, awsClient awsclient.Client, awsNotifications chan string, awsErrors chan string) error {
+func (r *ReconcileAccountClaim) rotateIAMUserCreds(reqLogger logr.Logger, awsClient awsclient.Client, claim *awsv1alpha1.AccountClaim, awsNotifications chan string, awsErrors chan string) error {
+
+	for _, user := range []string{osdManagedAdmin, osdManagedAdminSRE} {
+
+		getUserOutput, err := awsClient.GetUser(&iam.GetUserInput{UserName: aws.String(user)})
+		if err != nil {
+			getUserError := fmt.Sprintf("Could not find IAM user: %s", user)
+			awsErrors <- getUserError
+			return err
+		}
+
+		err = deleteAllAccessKeys(reqLogger, awsClient, user)
+		if err != nil {
+			delError := fmt.Sprintf("Failed deleting Access Keys for IAM user: %s", user)
+			awsErrors <- delError
+			return err
+		}
+
+		accessKeyOutput, err := account.CreateUserAccessKey(awsClient, getUserOutput.User)
+		if err != nil {
+			return err
+		}
+
+		secretName := claim.Spec.AccountLink + "-secret"
```

Comment:
We should use this function here https://github.com/openshift/aws-account-operator/blob/ad3cf6c45c8658ea1f12d63ab7c75fe2dc0390ab/pkg/controller/account/iam.go#L691-L695

Reply:
👍

Reply:
@jharrington22 This is a private func in the account controller. Should we move this to utils?

Reply:
sure makes sense